core/consensus/babe/impl/babe_impl.cpp

```diff
@@ -56,27 +56,57 @@ namespace kagome::consensus {
     BOOST_ASSERT(hasher_);
     BOOST_ASSERT(log_);
 
-    NextEpochDescriptor epoch_0_and_1_digest;
-    epoch_0_and_1_digest.randomness = genesis_configuration_->randomness;
-    epoch_0_and_1_digest.authorities =
-        genesis_configuration_->genesis_authorities;
-    epoch_storage_->addEpochDescriptor(0, epoch_0_and_1_digest);
-    epoch_storage_->addEpochDescriptor(1, epoch_0_and_1_digest);
+    NextEpochDescriptor init_epoch_desc;
+    init_epoch_desc.randomness = genesis_configuration_->randomness;
+    init_epoch_desc.authorities = genesis_configuration_->genesis_authorities;
+    [[maybe_unused]] bool init_epoch_desc_ok =
+        !!epoch_storage_->addEpochDescriptor(0, init_epoch_desc)
```

Comment:
what is !!

Reply:
if it is just to get bool, you can use .has_value() method from result

Reply:
Ok. Will be changed